webplugin/js/app/kommunicate-ui.js

```diff
@@ -974,8 +974,12 @@ KommunicateUI = {
         }
     },
     activateTypingField: function () {
-        !kommunicateCommons.checkIfDeviceIsHandheld() &&
+        if(kommunicate._globals.FocusOnTextArea) {
```

Comment:
if condition is not making any sense to me, please recheck 
try to explain what is 'if' doing and what is 'else' doing, that might give you a clarity

Reply:
Sir, I have changed the logic